python/triton/runtime/autotuner.py

```diff
@@ -88,10 +89,36 @@ def _post_hook(args, exception):
         self.base_fn = fn
         while not inspect.isfunction(self.base_fn):
             self.base_fn = self.base_fn.fn
-        self.num_warmups = warmup
-        self.num_reps = rep
-        import torch
-        self.use_cuda_graph = use_cuda_graph and torch.cuda.is_available()
```

Comment:
Hi @Jokeren, @int3,

Fields `self.num_warmups `, `self.num_reps` and `self.use_cuda_graph` are used by PyTorch to find out what parameters the autotuner was called with:

https://github.com/pytorch/pytorch/blame/5141ade8e30c64e873e14dcc8de233da45d15025/torch/_higher_order_ops/triton_kernel_wrap.py#L829

Can they be left until the corresponding parameters are removed from `__init__` signature?

Reply:
@int3 is driving the effort. It's up to him. I'm OK either way.